source/common/config/utility.h

```diff
@@ -217,16 +218,16 @@ class Utility {
    */
   template <class ProtoMessage, class Factory>
   static ProtobufTypes::MessagePtr
-  translateToFactoryConfig(const ProtoMessage& enclosing_message,
+  translateToFactoryConfig(absl::string_view extension_name, const ProtoMessage& enclosing_message,
```

Comment:
You should be able to use `enclosing_message.name()` instead of adding a parameter here?

Reply:
on a second thought, this should actually be `factory.name()` (if the factory doesn't have, add it), then you won't get deprecated name.

Reply:
Sadly not; `udp_listener_name` is the exception to this and that is in a v2 API. I wish we had caught that one in advance.

Reply:
@htuch `factory.name()` still work for that I guess?

Reply:
I can give that a shot, it will be a lot cleaner.

Reply:
Yep, that worked.